nixos/tests/agda.nix

```diff
@@ -0,0 +1,63 @@
+import ./make-test-python.nix ({ pkgs, ... }:
+
+let
+  testfile = pkgs.writeText "TestModule.agda" ''
+    module TestModule where
+  '';
+  # Make sure this does not get put in the nix store as an agda-lib file
+  mylibFile = pkgs.writeText "agda-lib-file" ''
```

Comment:
Another fix would be to use `pkgs.writeTextDir` I believe.

Reply:
That would put the file in a directory, solving the problem as well.

Reply:
It put's it in a directory but calls the directory `hash-mylib.agda-lib` so still breaks